tests/test_other.py

```diff
@@ -3529,12 +3521,16 @@ def test_warn_dylibs(self):
       warning = 'linking a library with `-shared` will emit a static object file'
       self.assertContainedIf(warning, err, suffix in shared_suffixes)
 
-  def test_symbol_map(self):
+  @parameterized({
+    'O2': [['-O2']],
+    # forced libs is ok, they were there anyhow
+    'O3': [['-O3']],
+  })
+  def test_symbol_map(self, opts):
```

Comment:
Is this part of this PR?

Reply:
Ah.. yes I refactored that test.  I can split it out.